guide/pyclass_parameters.md

```diff
@@ -33,7 +33,7 @@ struct MyClass {}
 struct MyClass {}
 ```
 
-[params-1]: https://docs.rs/pyo3/latest/pyo3/struct.PyAny.html
+[params-1]: https://docs.rs/pyo3/latest/pyo3/types/struct.PyAny.html
```

Comment:
This one can't, purely because this fragment is also included in `rustdoc` for `#[pyclass]`. I did that for re-use but it was not the best idea maybe.